components/src/preact/mutationComparison/mutation-comparison-venn.tsx

```diff
@@ -105,22 +110,48 @@ export const MutationComparisonVenn: FunctionComponent<MutationComparisonVennPro
             <div className='flex-1'>
                 <GsChart configuration={config} />
             </div>
-            <p class='flex flex-wrap break-words m-2'>{getSelectedMutationsDescription(selectedDatasetIndex, sets)}</p>
+            <p class='flex flex-wrap break-words m-2'>
+                <SelectedMutationsDescription
+                    selectedDatasetIndex={selectedDatasetIndex}
+                    sets={sets}
+                    sequenceType={sequenceType}
+                />
+            </p>
         </div>
     );
 };
 
 const noElementSelectedMessage = 'You have no elements selected. Click in the venn diagram to select.';
 
-function getSelectedMutationsDescription(
-    selectedDatasetIndex: number | null,
-    sets: ReturnType<typeof extractSets<string>>,
-) {
+type SelectedMutationsDescriptionProps = {
+    selectedDatasetIndex: number | null;
+    sets: ReturnType<typeof extractSets<string>>;
+    sequenceType: SequenceType;
+};
+
+const SelectedMutationsDescription: FunctionComponent<SelectedMutationsDescriptionProps> = ({
+    selectedDatasetIndex,
+    sets,
+    sequenceType,
+}) => {
     if (selectedDatasetIndex === null) {
         return noElementSelectedMessage;
     }
 
     const values = sets.datasets[0].data[selectedDatasetIndex].values;
     const label = sets.datasets[0].data[selectedDatasetIndex].label;
-    return `${label}: ${values.join(', ')}` || '';
-}
+    return (
+        <span>
+            {label}:{' '}
```

Comment:
```suggestion
           {`${label}: `}
```
Nitpick: I think this will do the same.